astropy/coordinates/sites.py

```diff
@@ -56,7 +56,7 @@ def __getitem__(self, site_name):
             close_names = get_close_matches(site_name, self._lowercase_names_to_locations)
             close_names = sorted(close_names, key=lambda x: len(x))
 
-            errmsg = ('Site "{0}" not in database. Use ``get_names()`` to see '
+            errmsg = ('Site "{0}" not in database. Use ``EarthLocation.get_site_names()`` to see '
```

Comment:
Ok I realize my mistake - this should actually say ``names`` (the property) and then indeed we could catch this in EarthLocation